### PR TITLE
remove unused zh locales directory in favor of zh_CN

### PR DESCRIPTION
**Pull request by adonesky1**
## Explanation
`zh_CN` is the correct language code for our Chinese translations. Crowdin was [mistakenly pushing them](https://consensys.slack.com/archives/C01DRUWDCFQ/p1663681058285809?thread_ts=1662710658.917469&cid=C01DRUWDCFQ) to the `zh` language code directory for a while. This was fixed as of this [PR](https://github.com/MetaMask/metamask-extension/pull/15910) and so `zh` translations are no longer updated and will not be used. I have verified that the `zh_CN` [locale file](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/zh_CN/messages.json) has since been correctly updated with all translations and is currently up to date minus any pending translations present in the current [Crowdin (github actions) PR](https://github.com/MetaMask/metamask-extension/pull/15999/files#diff-dd6394604a456b4fce777cf0a061c09765005fec76d283462c9bacb45804a18cR50).

## Pre-Merge Checklist

- [x] PR template is filled out
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] PR has been added to the appropriate release Milestone

### + If there are functional changes:

- [ ] Manual testing complete & passed
- [ ] "Extension QA Board" label has been applied

### CODE REVIEW

**Review by Gudahtt — DISMISSED** [review]
LGTM!

File: app/scripts/lib/get-first-preferred-lang-code.js [hunk 1]
```diff
@@ -37,9 +37,15 @@ export default async function getFirstPreferredLangCode() {
     .find(
       (code) =>
         existingLocaleCodes[code] !== undefined ||
-        existingLocaleCodes[code.split('-')[0]] !== undefined,
+        existingLocaleCodes[code.split('-')[0]] !== undefined ||
+        code === 'zh',
```
Comment (Gudahtt) [inline app/scripts/lib/get-first-preferred-lang-code.js:41]: Could we move this to a separate PR? This seems like a distinct change.

Comment (adonesky1) [reply app/scripts/lib/get-first-preferred-lang-code.js]: removed in latest commit and opened a new PR [here](https://github.com/MetaMask/metamask-extension/pull/16329)

**Review by Gudahtt — APPROVED** [review]

File: app/scripts/lib/get-first-preferred-lang-code.js (hunk 1 above)
Comment (Gudahtt) [reply app/scripts/lib/get-first-preferred-lang-code.js]: Thanks!

**Review by digiwand — APPROVED** [review]
